bin/main.ml

```diff
@@ -58,8 +65,19 @@ let gen_output ast f = function
       print_endline res
   | _ -> ()
 
+let process_pragmas (pragmas : Syntax.pragmas) : unit =
+  let process_global_pragma (k, v) =
+    match (k, v) with
+    | "PrintUsage", _ -> usage () |> print_endline
```

Comment:
Is PrintUsage particular? Or you just need a random example?
It seems rather unusual to require printing usage via a pragma

Reply:
it is a random example